jupyter_server/services/kernels/kernelmanager.py

```diff
@@ -738,28 +738,31 @@ async def wrapped_method(self, *args, **kwargs):
             # If the method succeeds, emit a success event.
             try:
                 out = await method(self, *args, **kwargs)
+                data = {
+                    "kernel_name": self.kernel_name,
+                    "action": action,
+                    "status": "success",
+                    "msg": success_msg.format(
+                        kernel_id=self.kernel_id, kernel_name=self.kernel_name, action=action
+                    ),
+                }
+                if self.kernel_id:
```

Comment:
This change is required to ensure that , it doesn't produce message with value 
`kernel_id : null` as that will fail with current schema. The current schema won't accept null value.